plugins/trx-plugin/src/main/java/io/qameta/allure/trx/TrxPlugin.java

```diff
@@ -104,14 +109,17 @@ protected void parseTestRun(final Path parsedFile, final RandomUidContext contex
 
     protected UnitTest parseUnitTest(final XmlElement unitTestElement) {
         final String name = unitTestElement.getAttribute(NAME_ATTRIBUTE);
+        final String fullName = unitTestElement.getFirst(TEST_METHOD_ELEMENT)
```

Comment:
Seems like it is would be better to call it `className`

Reply:
Fine with me.